src/styles/variables.scss

```
@@ -0,0 +1,8 @@
+$padding: 10px;
+$gap: 1px;
+$day-color: #eee;
+$day-color-hover: #ffbfcb;
+$day-height: 100px;
+$day-width: 100px;
```

Not need to duplicate variables with same values
```suggestion
$day-size: 100px;
```